README.md

```diff
@@ -117,10 +117,25 @@ Using multiple `id segment`s might seem like unnecessary overhead at first. Addi
 - Using the key/value syntax (`id segment key` `:` `id segment value`) with [well-known `id segment keys`](well-known-id-segment-keys.md) enables `PAC-ID` routing in generic contexts.
 
 ## PAC-ID Extension
+`PAC-ID`s can be extended with custom information after a * character. The * character and subsequent characters are not part of the `PAC-ID` and can therefore freely be used for one or multiple `extension`s. In order to facilitate parsing and rendering by systems like [PAC-ID resolver](https://github.com/ApiniLabs/PAC-ID-Resolver), `name` and `type` of the `extension` MUST be specified.
 
-`PAC-ID`s can be extended with custom information after a `*` character. As stated above, the `*` character and subsequent characters are not part of the `PAC-ID` and can therefore be used for custom extensions. 
 
-A well-known extension is the [T-REX](https://github.com/ApiniLabs/T-REX).
+Railroad diagram depicting the `extension`'s structure: ![Structure of extensions](images/railroad-diagram-extension.svg)
+
+| **Name** | **Meaning** | **Technical Requirements** |
+| :--- | :--- | :--- | 
+| `name` | Name of the extension | SHOULD indicate the purpose and scope of the extension|.
+| `type` | Identifies the format of `data` | SHOULD be a [well known extension types](/well-known-extension-types.md).|
+| `data` | Attached data | MUST be in the format, specified by `type`.
+
+
+
+An example of a PAC-ID with two extensions, one describing the calibration state of a balance, the other it's current measurements:
+```
+HTTPS://PAC.METTORIUS.COM/-MD/240:BAL500/21:210263*CAL$TREX/START$T.D:20240822+PF$T.B:T+17$T.D:20250822*CUR$TREX/TARE$MGM:2.5E2+ENV$KEL:293.15
```

Comment:
While this is obviously perfectly valid, I'd still recommend to always follow the same pattern of using the T-REX as a summary of the item referred to in the PAC for having coherent examples that would also scale to more complex scenarios where there would be actual datasets.